x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -39,32 +40,95 @@ export const ProcessEventDot = styled(
        */
       const projectionMatrix = useSelector(selectors.projectionMatrix);
       const [left, top] = applyMatrix3(position, projectionMatrix);
+
+      const [magFactorX] = projectionMatrix;
       const style = {
-        left: (left - 20).toString() + 'px',
-        top: (top - 20).toString() + 'px',
+        left: `${left}px`,
+        top: `${top}px`,
+        transform: `translateY(-50%) translateX(-50%) scale(${magFactorX})`,
       };
+
       return (
-        <span className={className} style={style}>
-          name: {event.data_buffer.process_name}
-          <br />
-          x: {position[0]}
-          <br />
-          y: {position[1]}
-        </span>
+        <svg
+          className={className}
+          style={style}
+          viewBox="-15 -5 30 10"
+          preserveAspectRatio="xMidYMid slice"
+          role="treeitem"
+          tabIndex={-1}
+          aria-level={event.data_buffer.depth}
+        >
+          <use
+            role="presentation"
+            xlinkHref={`#${SymbolIds.processNode}`}
+            x="-15.5"
+            y="-5"
+            width="31"
+            height="10"
+            opacity="1"
+          />
+          <text
+            x="0"
+            y="0"
+            textAnchor="middle"
+            dominantBaseline="middle"
+            fontSize="3"
+            fill={NamedColors.empty}
+            stroke={NamedColors.strokeBehindEmpty}
+            strokeWidth=".35"
+            paintOrder="stroke"
+          >
+            {event.data_buffer.process_name}
+          </text>
+
+          {magFactorX >= 1.75 ? (
+            <>
+              <text
+                x="0"
+                y="-2.1"
+                textAnchor="middle"
+                dominantBaseline="middle"
+                fontSize="1.25"
+                fill={NamedColors.empty}
+                stroke={NamedColors.strokeBehindEmpty}
+                strokeWidth=".25"
+                paintOrder="stroke"
+              >
+                Process
```

Comment:
this is likely not very useful -- since we're showing a process ancestry tree, everything is going to be a process... so just by definition, when you see one of these nodes isn't it going to be describing a process event?

Reply:
In context here: Yes, but from a UX standpoint it can be helpful to label things (and from a11y, sometimes it is even required), even when the label is small/de-emphasizes (e.g. this doesn't even appear until the camera sends a larger scale request for the scene).

In the larger picture: Pretty much everything visual in this PR is going change quickly towards either Lindsay's mocks or some other design direction based on customer feedback.